boxsdk/object/item.py

```diff
@@ -613,3 +621,94 @@ def get_collaborations(self, limit=None, marker=None, fields=None):
             fields=fields,
             return_full_pages=False,
         )
+
+    def add_classification(self, classification):
+        """
+        Applies metadata classification for the specified :class:`File` or :class:`Folder` object.
+
+        :param classification:
+            The classification to add to the :class:`File` or :class:`Folder`
+        :type classification:
+            `unicode`
+        :return:
+            The classification added to the :class:`File` or :class:`Folder.
+        :rtype:
+            `unicode`
+        """
+        classification_metadata = {
+            'Box__Security__Classification__Key': classification,
+        }
+        template_key = 'securityClassification-6VMVochwUWo'
+        metadata_classification = self.metadata('enterprise', template_key).create(classification_metadata)
+        return metadata_classification['Box__Security__Classification__Key']
+
+    def update_classification(self, classification):
+        """
+        Updates metadata classification for the specified :class:`File` or :class:`Folder` object.
+
+        :param classification:
+            The classification to add to the :class:`File` or :class:`Folder`
+        :type classification:
+            `unicode`
+        :return:
+            The classification updated on the :class:`File` or :class:`Folder.
+        :rtype:
+            `unicode`
+        """
+        classification_metadata = self.metadata('enterprise', 'securityClassification-6VMVochwUWo')
+        updates = classification_metadata.start_update()
+        updates.update('/Box__Security__Classification__Key', classification)
```

Comment:
Does this always work?  I think there are two possible cases for updating the metadata instance:

1. The instance already has a value for the classification, so update is correct
2. The instance exists, but does not already have the `Box__Security__Classification__Key` field set — like when the user has used the webapp to remove classification — so update may not work (you may need add instead)

Reply:
@mattwiller in this case do we try update first and if that throws we catch and do add or do we just completely do add instead?